image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -82,6 +82,9 @@ spec:
         - name: ROX_TELEMETRY_STORAGE_KEY_V1
           value: {{ ._rox.central.telemetry.storage.key | quote }}
         {{- end }}
```

Comment:
Is the `{{ end }} {{ else }}` correct template syntax? https://helm.sh/docs/chart_template_guide/control_structures/#ifelse does not have intermittent `{{ end }}`

Reply:
this `{{ end }}` is closing another `{{ if }}`. I'll put a clarifying comment.